README.md

```diff
@@ -128,7 +128,7 @@ How to register Monster-UI apps.
 3. You need to 'register' these apps
 
 ```sh
-docker cp monster-ui.kazoo:/usr/share/nginx/html/apps apps
```

Comment:
The proper folder is /usr/share/nginx/html/dist/apps (these are compiled versions of src/apps).